src/app/views/calendar.html

```diff
@@ -12,9 +12,13 @@ <h3 class="header">
         </button>
       </li>
     </h3>
+    <div class="week">
+      Ført {{hours}} av {{availableHours}} timer denne uken
+    </div>
+
     <ul class="week">
         <li class="day" ng-repeat="day in week" ng-click="select(day.date)" title="{{day.date}}" ng-class="{selected: isSelected(day.date)}">
-            <span class="dayInMonth" ng-class="{sunday: $index == 6}">{{day.dayInMonth}}</span>
+            <span class="dayInMonth" ng-class="{holiday: $index === 6 || $index === 5 || holidays.indexOf($index) > -1 }">{{day.dayInMonth}}</span>
```

Comment:
Maybe use `holidays.includes($index)` instead? The current method seems a bit… indirect.